### PR TITLE
Adds React full width resizable table example

### PR DESCRIPTION
**Pull request by juhanakristian**
This PR adds a React example similar to the old [full-width resizable table example](https://github.com/TanStack/table/tree/main/old-examples/full-width-resizable-table)

The new example doesn't have the row selection from the old example.

### CODE REVIEW

File: examples/react/full-width-resizable-table/src/main.tsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+              </tr>
+            )
+          })}
+        </tbody>
+      </table>
+    </div>
+  )
+}
+
+ReactDOM.render(
```
Comment (subbarao) [inline examples/react/full-width-resizable-table/src/main.tsx:153]: can you upgrade to react 18.2.0
https://github.com/TanStack/table/blob/main/examples/react/basic/src/main.tsx#L169

File: examples/react/full-width-resizable-table/yarn.lock [hunk 2]
```diff
@@ -0,0 +1,323 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@faker-js/faker@6.3.1":
```
Comment (subbarao) [inline examples/react/full-width-resizable-table/yarn.lock:5]: don't add yarn.lock. none of the other examples added yarn.lock.

File: examples/react/full-width-resizable-table/package.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
+    "start": "vite"
+  },
+  "dependencies": {
+    "@faker-js/faker": "6.3.1",
+    "@tanstack/react-table": "8.0.0",
+    "faker": "^6.6.6",
```
Comment (subbarao) [inline examples/react/full-width-resizable-table/package.json:14]: you don't need faker dependency as you already have faker-js.

File: examples/react/full-width-resizable-table/package.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    "react": "17.0.2",
+    "react-dom": "17.0.2"
+  },
+  "devDependencies": {
+    "@rollup/plugin-replace": "^4.0.0",
```
Comment (subbarao) [inline examples/react/full-width-resizable-table/package.json:19]: add faker type as dev dependency.
https://github.com/TanStack/table/blob/main/examples/react/column-ordering/package.json#L19

**Comment by tannerlinsley** [comment]
Still working on this?

**Comment by juhanakristian** [comment]
Yeah, sorry for taking this long. I’m also migrating it to useReactTable.

File: examples/react/full-width-resizable-table/src/main.tsx (hunk 1 above)
Comment (juhanakristian) [reply examples/react/full-width-resizable-table/src/main.tsx]: Example is now using React 18.2.0

File: examples/react/full-width-resizable-table/yarn.lock (hunk 2 above)
Comment (juhanakristian) [reply examples/react/full-width-resizable-table/yarn.lock]: yarn.lock has been removed from the PR

File: examples/react/full-width-resizable-table/package.json (hunk 3 above)
Comment (juhanakristian) [reply examples/react/full-width-resizable-table/package.json]: faker removed from deps

File: examples/react/full-width-resizable-table/package.json (hunk 4 above)
Comment (juhanakristian) [reply examples/react/full-width-resizable-table/package.json:18]: @types/faker has been added to devDependencies

**Pull request merged**